jtune.py

```diff
@@ -1777,6 +1780,20 @@ def get_jmap_data(pid=None, procdetails=None):
 # Main
 user = getpass.getuser()
 
+
+def get_latest_rotated_log_file(gc_log_file, uses_file_rotation):
```

Comment:
Rework this so that it retrieves the gc log file path and filename within the function, and that gets assigned to the gc_log_file variable in __main__. This function should accept the parameters 'gc_file_rotation' boolean, and 'proc_details' dictionary. Let me know if this doesn't make sense...